pkg/controller/servicemesh/controlplane/controller_test.go

```diff
@@ -91,7 +91,7 @@ func TestCyclicTemplate(t *testing.T) {
 	reconciler := reconcileControlPlane.getOrCreateReconciler(&v1.ServiceMeshControlPlane{})
 	reconciler.Log = log.WithValues()
 
-	_, err := reconciler.renderSMCPTemplates(v1.ControlPlaneSpec{Template: "visited"}, map[string]struct{}{"visited": {}})
+	_, err := reconciler.recursivelyApplyTemplates(v1.ControlPlaneSpec{Template: "visited"}, map[string]struct{}{"visited": {}})
```

Comment:
@brian-avery Was this change required by the grafana work?

Reply:
@knrc No, this change wasn't required by the Grafana work. If you'd like I'll move it to another PR. When going through CR comments we changed method names and this changes the method names in the unit tests to match the actual function name.

Reply:
We don't run unit tests so there's nothing that actually cares if this change is in there.

Reply:
Yes please, we should move this to another PR

Reply:
Removed

Reply:
this change is now in: https://github.com/Maistra/istio-operator/pull/214